app/src/test/java/tech/ula/viewmodel/MainActivityViewModelTest.kt

```diff
@@ -83,7 +86,7 @@ class MainActivityViewModelTest {
         mainActivityViewModel.waitForPermissions(appToContinue = selectedApp)
         mainActivityViewModel.permissionsHaveBeenGranted()
         runBlocking {
-            delay(1)
+            delay(2)
```

Comment:
We should convert this `2` seconds value to a variable.  

1) It'll add clarity to what it does and 
2) we'll only have to change it in one spot if we need to adjust it in the future.